phylogenetic/rules/annotate_phylogeny.smk

```diff
@@ -1,32 +1,51 @@
 """
 This part of the workflow creates additonal annotations for the phylogenetic tree.
 
-REQUIRED INPUTS:
-
-    metadata            = data/metadata.tsv
-    prepared_sequences  = results/prepared_sequences.fasta
-    tree                = results/tree.nwk
-
-OUTPUTS:
-
-    node_data = results/*.json
-
-    There are no required outputs for this part of the workflow as it depends
-    on which annotations are created. All outputs are expected to be node data
-    JSON files that can be fed into `augur export`.
-
-    See Nextstrain's data format docs for more details on node data JSONs:
-    https://docs.nextstrain.org/page/reference/data-formats.html
-
-This part of the workflow usually includes the following steps:
-
-    - augur traits
-    - augur ancestral
-    - augur translate
-    - augur clades
-
 See Augur's usage docs for these commands for more details.
 
-Custom node data files can also be produced by build-specific scripts in addition
-to the ones produced by Augur commands.
 """
+
+rule ancestral:
+    """Reconstructing ancestral sequences and mutations"""
+    input:
+        tree = "results/tree.nwk",
+        alignment = "results/aligned.fasta"
+    output:
+        node_data = "results/nt_muts.json"
+    log:
+        "logs/ancestral.txt",
+    benchmark:
+        "benchmarks/ancestral.txt"
+    params:
+        inference = config["ancestral"]["inference"]
+    shell:
+        """
+        augur ancestral \
```

Comment:
Related to your recent work on root vs reference sequences, why not use `--root-sequence` so that when/if used as an ad-hoc nextclade dataset we get a better reference?

Reply:
Done in [571ef05](https://github.com/nextstrain/rabies/pull/3/commits/571ef05b4f328ec947310b2e8cd9eda89f58a96e)